string name
# uint32 id 		# serial number of message
# float64 stamp 	# time of measurement
pid3 gains
pid3 errors

float64 state
float64 input
float64 dinput
float64 output
float64 limit
float64 time_constant

